packages/@webex/plugin-cc/src/types.ts

```diff
@@ -247,7 +247,9 @@ export type AgentLogin = {
   loginOption: LoginOption;
 };
 
-export type AgentDeviceUpdate = Pick<AgentLogin, 'loginOption' | 'dialNumber'>;
+export type AgentDeviceUpdate = Pick<AgentLogin, 'loginOption' | 'dialNumber'> & {
+  teamId?: string;
```

Comment:
I think we should stick to asking for a teamId all the time. Just to avoid confusions. The code to pick existing team can stay there but we need not mark it optional.